File: core/src/com/mygdx/game/entity/BaseResult.java
package com.mygdx.game.entity;

public class BaseResult {
    private int type = 0;
    public static final int EVENT_BUY_LAND = 1;
    public static final int EVENT_BUILD_LAND = 2;
    public static final int EVENT_PAY_LAND = 3;

    public BaseResult(int type) {
        this.type = type;
    }

    public int getType() {
        return type;
    }

    public void setType(int type) {
        this.type = type;
    }
}

File: core/src/main/kotlin/com/rich/diy/game/entity/BaseResult.kt
package com.rich.diy.game.entity

open class BaseResult(type: Int) {
    var type = 0

    companion object {
        const val EVENT_BUY_LAND = 1
        const val EVENT_BUILD_LAND = 2
        const val EVENT_PAY_LAND = 3
    }

    init {
        this.type = type
    }
}